src/routes/DesignSystemApp.svelte

```diff
@@ -107,9 +109,55 @@
     </ONSCollapsible>
   </DesignSystemPanel>
 
-  <DesignSystemPanel title="Input" code={`<ONSTextField id="text-field">Text field</ONSTextField>`}>
-    <ONSTextField id="text-field">Text field</ONSTextField>
+  <DesignSystemPanel
+    title="Input"
+    code={`<ONSTextField
+      id="text-field-1"
+      type="text"
+      bind:textFieldValue
+      labelText="Text field"
+      accessiblePlaceholder
+      onInput={(textFieldValue) => console.log("Input user value: ", textFieldValue)}
+      onChange={(textFieldValue)=>console.log("Displays what user is typing every time he hits the return key: ",textFieldValue)}
```

Comment:
Please make language gender neutral so 
`Displays what user is typing every time they hit the return key`

Reply:
I've just updated the comments.